microshift_updating/microshift-update-options.adoc

```diff
@@ -14,13 +14,13 @@ include::snippets/microshift-update-paths-snip.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-Updates of {microshift-short} from one minor version to the next must be in sequence. For example, you cannot update from 4.14 to 4.16. You must update 4.14 to 4.15 and so on.
+Updates of {microshift-short} are supported with a maximum of two minor versions. For example, you can update from 4.14 to 4.16 in a single step. Updating 4.14 to 4.15 first is not required.
 ====
 
 [id="microshift-update-options-standalone-updates_{context}"]
 == Standalone {microshift-short} updates
 
-You can update {microshift-short} without reinstalling your applications. {op-system} or {op-system-ostree} updates are also not required to update {microshift-short}, as long as the existing operating system is compatible with the new version of {microshift-short} that you want to use.
+You can potentially update {microshift-short} without reinstalling your applications and Operators. {op-system} or {op-system-ostree} updates are only required to update {microshift-short} if the existing operating system is not compatible with the new version of {microshift-short} that you want to use.
```

Comment:
```suggestion
You can potentially update {microshift-short} without reinstalling your applications and Operators. {op-system} or {op-system-ostree} updates are only required before updating {microshift-short} if the existing operating system is not compatible with the new version of {microshift-short} that you want to use.
```

Reply:
Under what conditions can update without reinstalling apps or operators. The second sentence doesn't seem to answer this question. (Or, maybe it does and I am ignorant here.).

Reply:
Separate thoughts, changed the structure a bit. In reality, we can'r answer that question here, customer has to figure that out based on custom apps and whatnot.